app/views/balrog/gate.html

```diff
@@ -0,0 +1,26 @@
+<!doctype html>
+<html lang=en>
+  <head>
+    <meta charset=utf-8>
+    <title>Login</title>
+    <link rel="stylesheet" media="screen" href="/assets/balrog/gate.css">
```

Comment:
You are not going to be able to use assets like this any more. These are in the asset pipeline, so they are fingerprinted. It'll work in dev, and fail in production.

I suggest you inline the CSS & the PNG in this file. You can inline the PNG by using a data URI (https://en.wikipedia.org/wiki/Data_URI_scheme)